spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/spring/data/firestore/mapping/FirestorePersistentEntityImpl.java

```diff
@@ -74,4 +82,15 @@ public FirestorePersistentProperty getIdPropertyOrFail() {
 			return collectionName;
 		}
 	}
+
+	@Override
+	public void addPersistentProperty(FirestorePersistentProperty property) {
+		super.addPersistentProperty(property);
+		if (property.findAnnotation(UpdateTime.class) != null) {
+			if (property.getActualType() != Timestamp.class) {
+				throw new FirestoreDataException("@UpdateTime annotated field should be of com.google.protobuf.Timestamp type");
```

Comment:
Would it be possible to use our converters instead of requiring `com.google.protobuf.Timestamp`??

Reply:
we don't have any converters, we use client library for converting

Reply:
Does the client library allow any other date types that can be automatically converted to `com.google.protobuf.Timestamp`?